cmd/docs/docs_test.go

```diff
@@ -0,0 +1,107 @@
+package docs
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewDocsCommand(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Docs generate using default path", func(t *testing.T) {
+		t.Parallel()
+		cmd := NewDocsCommand()
+		err := cmd.Execute()
+		require.NoError(t, err)
+
+		// Check if a Markdown file was generated
+		files, err := os.ReadDir(DefaultPath)
+		assert.NoError(t, err)
+		assert.NotEmpty(t, files)
+
+		markdownFileFound := false
+
+		for _, file := range files {
+			if strings.HasSuffix(file.Name(), ".md") {
+				markdownFileFound = true
```

Comment:
For the docs generation, I'm keeping it simple. I'm only checking if there was markdown generated as the text would change over time as the CLI evolves and I think mocking what `doc.GenMarkdownTree` does might be overkill.